openlibrary/templates/about/team_member.html

```diff
@@ -0,0 +1,59 @@
+<link
+	href="$static_url('build/page-teamMember.css')"
+	rel="stylesheet"
+	type="text/css" />
```

Comment:
Eventually we'll want to conditionally load the style sheet based on the page we're on (I think templates/lib/ or templates/site may have the relevant logic? I have to check)

Reply:
Fixed!